python/lookout/sdk/pb.py

```diff
@@ -0,0 +1,14 @@
+"""
+Re-exporting all grpc related classes/functions for cleaner API
+"""
+
+from .service_analyzer_pb2_grpc import \
+    AnalyzerServicer, add_AnalyzerServicer_to_server as add_analyzer_to_server
+
+from .event_pb2_grpc import *
+from .event_pb2 import *
+from .grpc import *
```

Comment:
Should we separate our code from autogenerated one? I would rather explicitly import this:

```python
from lookout.sdk.pb import AnalyzerServicer
from lookout.sdk.grpc import to_grpc_address
```

Reply:
done in https://github.com/src-d/lookout-sdk/pull/43/commits/03c416667581585f56c5412fc2348a95ef0aa722